src/canvas/DesktopElement.ts

```diff
@@ -428,12 +443,12 @@ export default class GmfDesktopCanvas extends BaseElement {
   }
 
   updated(): void {
-    new ResizeObserver(() => {
-      if (this.map_) {
-        this.map_.updateSize();
-        this.map_.renderSync();
-      }
-    }).observe(this.shadowRoot.getElementById('gmf-app-map'));
+    const mapElement = this.shadowRoot.getElementById('gmf-app-map');
```

Comment:
You can declare a property

```ts
@query('gmf-app-map')
gmfAppMap: Element;
```

Then do `if (this.gmfAppMap !== this.mapElement_) ...`